test/test_unary_ufuncs.py

```diff
@@ -1094,6 +1094,48 @@ def test_mish(self, device, dtype):
             rtol=rtol,
         )
 
+    @dtypes(torch.complex64, torch.complex128)
+    @onlyCPU
+    def test_log1p_complex(self, device, dtype):
```

Comment:
Actually, this may something that may be of interest in general for unary and binary ufuncs. See `test_unary_ufuncs.py`. Perhaps someone from intel may be interested in implementing this? @jgong5 The reference for the test can be either numpy or a non-contiguous tensor.

Reply:
> Actually, this may something that may be of interest in general for unary and binary ufuncs. See `test_unary_ufuncs.py`. Perhaps someone from intel may be interested in implementing this? @jgong5 The reference for the test can be either numpy or a non-contiguous tensor.

You are talking about a way of test the vector and non-vector implementation in general, right? I noticed that a lot of tests in `test_unary_ufuncs.py` already use a big size and should hit the vector path already (even though the scalar path might not be covered then). Some tests use smaller size and the vector path is not covered though. This would need a more thorough review to know the current status. We can probably make sure all the implementations can trigger the vector path and then control the `ATEN_CPU_CAPABILITY` env var to cover both vector and non-vector paths with the same test code. cc @sanchitintel and @mingfeima for their opinions.

Reply:
So, what I've seen is that there are some one-offs tests for some ops, but we could potentially have a dedicated test that tests the vectorised path against the non-vectorised explicitly. The plan to test depending on the CPU capabilities sounds great as well!